modules/uncertainty/R/generate_OAT_SA_design.R

```diff
@@ -0,0 +1,123 @@
+#' Generate One-At-a-Time (OAT) input design for sensitivity analysis
+#'
+#' Creates an input design matrix for sensitivity analysis where non-parameter
+#' inputs (met, IC, soil, etc.) are held constant while parameters vary
+#' one-at-a-time across quantiles. This differs from ensemble design where
+#' all inputs vary together.
+#'
+#' @param settings PEcAn settings object
+#' @param sa_samples Optional. Pre-loaded SA samples from samples.Rdata.
```

Comment:
This argument suggests a fundamental design misunderstanding. sa_samples should be *generated by* this function, not read by it. I'm OK with taking this as an optional input if one wants to reuse a previous sa_samples, but if it's not provided it needs to be generated